essentials/tools.mdx

```diff
@@ -505,17 +505,34 @@ These tool actions require setting a special response type.
 |------|------|
 | hang-up | Terminates the call. In addition to having Ultravox end the call after [periods of user inactivity](/api-reference/calls/overview#inactivitymessages), your custom tool can end the call. |
 | new-stage | Creates a new call stage. See [here](/guides/callstages) for more. |
+| agent_reaction | Determines how the agent reacts after a tool call. See [below](#controlling-agent-responses-to-tools) for more. |
 
 How you set the response type depends on if you are using a server/HTTP or a client tool:
 
 **Server/HTTP Tools**
 
-Server tools must respond with the `X-Ultravox-Response-Type` header set to either `hang-up` or `new-stage`
+Server tools must respond with the `X-Ultravox-Response-Type` header set to either `hang-up` or `new-stage`.
 
 **Client Tools**
 
 For client tools, set `responseType="hang-up"` or `responseType="new-stage"` on your `ClientToolResult` object.
 
+### Controlling Agent Responses to Tools
+By default, LLM inference runs immediately after a tool call. This has the effect of the agent speaking. This is typically the desired behavior for tools that gather information. For example, for a tool that gathers information, the agent might say something like, "Great, I've submitted your information."
```

Comment:
IMO "I've submitted your information" is the opposite of gathering information -- I think things like KB searches or customer account lookup would be better examples.